cmd/buildah/import.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/containers/image/storage"
+	"github.com/containers/storage/pkg/chrootarchive"
+	"github.com/pkg/errors"
+	"github.com/projectatomic/buildah"
+	"github.com/urfave/cli"
+)
+
+var (
+	importFlags = []cli.Flag{
+		cli.StringSliceFlag{
+			Name:  "change, c",
+			Usage: "Apply specified Dockerfile instructions while importing the image",
+		},
+		cli.StringFlag{
+			Name:  "message, m",
+			Usage: "Set commit message for imported image",
+		},
+		cli.StringFlag{
+			Name:  "signature-policy",
+			Usage: "`pathname` of signature policy file (not usually used)",
+		},
+	}
+	importCommand = cli.Command{
+		Name:        "import",
+		Usage:       "Create an empty filesystem image and import the contents of the tar archive.",
+		Description: `Supported archive formats: ().tar, .tar.gz, .tgz, .bzip, .tar.xz, .txz), then optionally tag it.`,
```

Comment:
Interesting, thanks for checking.  Not sure about the errors below.  Should "tar archive file" just be shortened to "archive file"?